sdks/python/setup.py

```diff
@@ -104,6 +104,7 @@ def get_version():
 REQUIRED_PACKAGES = [
     'avro>=1.8.1,<2.0.0; python_version < "3.0"',
     'avro-python3>=1.8.1,<2.0.0; python_version >= "3.0"',
+    'collapsing-thread-pool-executor>="2018.6"',
```

Comment:
Couple of questions about the dependency:
- Could we add an upper bound? (e.g. <2019)
- This does not look like a very used library with total 10 commits and no updates in the past year. Is there an alternative?

Reply:
We could add one but it would be arbitrary. WDYT?

I looked through Pypi and could not find another one, I looked at about 40 libraries that mentioned threadpool and this one was the best since it:
* mirrored the ThreadPoolExecutor API that is part of Python
* didn't have any additional dependencies
* I read the code and it was based upon the ThreadPoolExecutor implementation that is part of Python

Reply:
Sounds good to me. Thank you for the detailed due diligence.